internal/connectors/engine/engine.go

```diff
@@ -368,6 +368,28 @@ func (e *engine) UpdateConnector(ctx context.Context, connectorID models.Connect
 		otel.RecordError(span, err)
 		return err
 	}
+
+	_, err = e.temporalClient.ExecuteWorkflow(
```

Comment:
I wonder if we want to have a "hook" here to easily add future migrations, if not this will grow and grow.
Ideally we'd want to keep track of what migration was ran etc.

Maybe Temporal already have a migration path? Maybe worth asking.